src/instrumentsscene/view/systemobjectslayertreeitem.cpp

```diff
@@ -181,28 +177,6 @@ void SystemObjectsLayerTreeItem::onScoreChanged(const mu::engraving::ScoreChange
     }
 }
 
-void SystemObjectsLayerTreeItem::onVisibleChanged(bool isVisible)
-{
-    if (m_ignoreVisibilityChanges || m_systemObjectGroups.empty()) {
```

Comment:
Could you please remove the m_ignoreVisibilityChanges flag? And this code:

<img width="375" alt="Screenshot 2025-02-26 at 16 12 51" src="https://github.com/user-attachments/assets/ffa70305-98dc-4496-9699-f219deb2defb" />